index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+</head>    
+<body>
+    <nav class="container">
+		<div class="navContainer">
+			<div class="banner">
+				<h1>Luke Chaston</h1>
+				<h2>Web Developer | Programmer </h2>
+			</div>
+			<section class="buttonContainer">
+				<div class="button"><a href="#">About Me</a></div>
+				<div class="button"><a href="#">Portfolio</a></div>
+				<div class="button"><a href="#">Contact</a></div>
+			</section>
+				</nav>
+		</div>
+		<div class="biography">
+			    <div class="photo">
+				<p>Where the photo will go.</p>
+			    </div>
+			      <section class="about">
+				  <h1>About Me</h1>
+				  <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."</p>
+				  <h1>Skills</h1>
+				  <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."</p>
+				  <h1>This and That</h1>
+				  <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."</p>
+			  </section>
+			 </div>
+			 
+			<div class="portfolio">
+				
+				<div class="grid">
+					<h1>Portfolio</h1>
+						<h3>These are some of the websites and designs I have worked on.</h3>
+			        <section class="column">
+					<div class="pilot">
```

Comment:
doesn't look like you need pilot class, you haven't linked to it anywhere on the CSS